docs/usage/shoot_settings/default_seccomp_profile.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Default Seccomp Profile
+description: Enable the use of `RuntimeDefault` as the default seccomp profile through `spec.kubernetes.kubelet.seccompDefault`
```

Comment:
I would avoid adding the field as kubelet settings can be provided globally (https://github.com/gardener/gardener/blob/98fb871eb6045f69715d9f8b39dd84f9637f1fa0/example/90-shoot.yaml#L237, `spec.kubernetes.kubelet`) and they can be also provided per worker pool (https://github.com/gardener/gardener/blob/98fb871eb6045f69715d9f8b39dd84f9637f1fa0/example/90-shoot.yaml#L79, `spec.provider.workers[].kubernetes.kubelet`).